src/uniprotkb/components/entry/DiseaseAndDrugsSection.tsx

```diff
@@ -94,9 +96,35 @@ const DiseaseAndDrugsSection = ({
         features={data.featuresData}
         sequence={sequence}
       />
-      <LazyComponent fallback="Variants" rootMargin="50px">
-        <VariationView primaryAccession={primaryAccession} title="Variants" />
-      </LazyComponent>
+      {hasImportedVariants ? (
+        <section>
+          <h3>Variants</h3>
+          <div className={styles.variants}>
+            <img src={variantsImg} width="1944" height="1024" alt="" />
+            <div>
+              <p>
+                We are now providing the &quot;Disease & Variants&quot; viewer
+                on its own tab.
+              </p>
+              <p>
+                The viewer provides UniProt and other sources of variants from
+                ClinVar, dbSNP, etc.
+              </p>
+              <p>
+                <Link
+                  to={getEntryPath(
+                    Namespace.uniprotkb,
+                    primaryAccession,
+                    TabLocation.VariantsViewer
+                  )}
+                >
+                  Go to variants tabs <FullViewIcon width="0.75em" />
+                </Link>
+              </p>
+            </div>
+          </div>
+        </section>
+      ) : null}
```

Comment:
This results in an empty section:

<img width="316" alt="Screenshot 2023-04-11 at 17 07 55" src="https://user-images.githubusercontent.com/4357962/231223255-26dadb09-b16b-48e6-a799-1fc134afa1bd.png">

While we used to display:
```
No variation information available for A0A024QZA8
```

Could we redisplay this or maybe even not render the section at as we have will know based on the previous `HEAD` request?